pkgs/development/tools/rubocop/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+bundlerEnv rec {
+  name = "rubocop-${version}";
+  version = "0.47.0";
+
+  inherit ruby;
+
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
```

Comment:
@Mic92 @zimbatm I can make a separate PR for that if it makes sense. I suspect there will be quite a lot of deprecation warnings to start with, though.